### PR TITLE
Cut release 1.3.0

### PR DESCRIPTION
**Pull request by fabxc**
@beorn7

### CODE REVIEW

**Comment by brancz** [comment]
lgtm 👍 let's get it out there

**Comment by beorn7** [comment]
Can we have https://github.com/prometheus/prometheus/pull/2143 in there to fix that nasty bug?

**Comment by beorn7** [comment]
i.e. you can still trigger the same panic with a negative staleness delta on the command line, and there might be other paths to the range query that would still allow the inverted time range.

**Comment by fabxc** [comment]
#2143 is in there, see last line in changelog.
I merged that back into master without PR as we obviously want it in there anyway.

**Comment by beorn7** [comment]
You merged #2142 but no #2143 with contains the "proper" fix.

**Comment by fabxc** [comment]
Ah, I didn't even see that one, thanks. I'll blame it on the jetlag :)

**Comment by fabxc** [comment]
@beorn7 your PR is merged, changes merged back into master and release-1.3 respectively.

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ This is a breaking change to the Kubernetes service discovery.
 * [ENHANCEMENT] Add Azure SD metrics.
 * [ENHANCEMENT] Add fuzzy search to `/graph` textarea.
 * [ENHANCEMENT] Always show instance labels on target page.
+* [BUGFIX] Validate query end time is not before start time.
```
Comment (beorn7) [inline CHANGELOG.md]: Perhaps add that negative staleness deltas are now rejected when provided no the command line? (In principle, somebody might have done so, and now the server wouldn't start up anymore.)

**Comment by beorn7** [comment]
Thanks. Just one comment about the changelog.

**Comment by fabxc** [comment]
Theoretically a breaking change – but as it caused the server to panic, categorizing it as bug fix seems fine.

**Comment by beorn7** [comment]
Exactly.

:+1:

**Pull request merged**